view/adminhtml/web/vue/FieldType/Autocomplete.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <div class="admin__field field field-title">
+        <label class="label admin__field-label">
+            {{ label }}
+        </label>
+        <div class="admin__field-control control">
+            <v-select
+                v-model="item.content"
+                :options="fieldOptions"
+                :placeholder="placeHolder"
+            >
+            </v-select>
+        </div>
+        <div class="selected-option">
+            <div class="selected-option__label">
+                {{ description }}
+            </div>
+            <div class="selected-option__value">
+                <span v-if="selectedId">
+                    {{ selectedId }}
+                </span>
+                <span v-else>
+                    {{ placeHolder }} ⇡
```

Comment:
Is that `⇡` here for purpose?

Reply:
yes it was in the old design to direct user to select a value from the above field (category, page, block)